src/metabase/lib/column_group.cljc

```diff
@@ -72,9 +72,9 @@
         (lib.metadata.calculation/display-info query stage-number table))
       (when-let [card (some->> (:source-card stage) (lib.metadata/card query))]
         (lib.metadata.calculation/display-info query stage-number card))
-      ;; for multi-stage queries return an empty string (#30108)
+      ;; multi-stage queries (#30108)
       (when (next (:stages query))
-        {:display-name ""})
+        {:display-name "Summaries"})
```

Comment:
Is this the corresponding stage always a summarization?

Reply:
You can't create multiple stage queries without summarization.